code/modules/mob/living/carbon/human/human.dm

```diff
@@ -32,8 +32,6 @@
 
 	create_reagents(330)
 
-	prev_gender = gender // Debug for plural genders

```

Comment:
Right. You want me to re-add it or is this edge case enough that it's not worth really keeping? I've...not really ever heard of someone having a gender they're not supposed to have, before.